Pages/LanguagePage.cs

```diff
@@ -0,0 +1,113 @@
+﻿using OpenQA.Selenium;
+using SpecFlowMars.Utilities;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SpecFlowProjectMars2.Pages
+{
+    public class LanguagePage : CommonDriver
+    {
+        public LanguagePage(IWebDriver driver) : base(driver)
+        {
+        }
+        public LanguagePage() { }   
```

Comment:
no needed by default you will get a default constructor, you will only do this if you are doing something in the constructor e.d
public LanguagePage(IWebDriver driver) {
 this. driver = driver
 }

please also remove this 
public LanguagePage(IWebDriver driver) : base(driver)
        {
        }
 this makes no sense